packages/website/src/MarkdownEditor/Tiptap/EditorMenu.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+
+import { Editor } from "@tiptap/react";
+import "./EditorMenu.scss";
+
+export interface Props {
+  editor: Editor | null;
+}
+
+export const EditorMenu: React.FC<Props> = ({ editor }) => {
+  if (!editor) {
+    return null;
+  }
+
+  return (
+    <div className="EditorMenu">
```

Comment:
Do they not have a basic prebuilt menu component?

Reply:
Apparently not! This was their "complete setup" https://tiptap.dev/installation/react#5-the-complete-setup-optional

Reply:
Does look like we can add some cool things easily though: https://tiptap.dev/examples/menus